### PR TITLE
Setup CI and scripts to prepare production deployments

### PR DESCRIPTION
**Pull request by giacomolicari**
Add script to prepare for deployments to production, add steps in CI to run prepare deploy.

### CODE REVIEW

**Review by anxolin — COMMENTED** [review]
Nice! thanks @giacomolicari 

I think we might be missing the Release trigger: https://github.com/gnosis/gp-swap-ui/blob/develop/.github/workflows/ci.yml#L9 

I think the `on` condition, we can leave it exactly as in `gp-swap-ui`

File: .github/scripts/prepare_production_deployment.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ev
+
+# Only:
+# - Tagged commits
+# - Security env variables are available.
+if [ -n "$VERSION_TAG" ] && [ -n "$PROD_DEPLOYMENT_HOOK_TOKEN" ] && [ -n "$PROD_DEPLOYMENT_HOOK_URL" ]
+then
+  curl --silent --output /dev/null --write-out "%{http_code}" -X POST \
```
Comment (alfetopito) [inline .github/scripts/prepare_production_deployment.sh:10]: Where does `"%{http_code}"` come from?
Oh, that's from the response from the POST, right?

**Review by alfetopito — APPROVED** [review]

File: .github/scripts/prepare_production_deployment.sh (hunk 1 above)
Comment (giacomolicari) [reply .github/scripts/prepare_production_deployment.sh:10]: You're right, it is the http code from the POST response, for debugging purposes

**Comment by giacomolicari** [comment]
> Nice! thanks @giacomolicari
> 
> I think we might be missing the Release trigger: https://github.com/gnosis/gp-swap-ui/blob/develop/.github/workflows/ci.yml#L9
> 
> I think the `on` condition, we can leave it exactly as in `gp-swap-ui`

Yes, I understand `on release` would automatically contain `on tags`, I will update it :)

**Comment by alfetopito** [comment]
@anxolin Anything preventing this to be merged?

**Review by anxolin — APPROVED** [review]
> @anxolin Anything preventing this to be merged?

All good 👌

The missing part u already did in the show native token PR.

**Pull request merged**